### PR TITLE
Move action buttons and hide restorer button based on permissions

### PR DESCRIPTION
**Pull request by susiecoleman**
There is now wider access to composer restorer but the limitations on the number of people who can restore has remained the same. The restore button should not be available to people who cannot restore content. Buttons were also moved from snapshot list to main section to simplify the layout of the snapshot list.

Moved JSON button and restore button:
restorer - permission to restore

Hide restore button when user cannot restore:
restorer - no permission to restore

JSON button now allows the user to toggle between json and text:
restorer - toggle text

### CODE REVIEW

File: public/javascripts/app/controllers/SnapshotContentCtrl.js [hunk 1]
```diff
@@ -11,9 +11,22 @@ SnapshotContentCtrlMod.controller('SnapshotContentCtrl', [
...
+    $scope.displayButtonLabel = "JSON";
+    $scope.canRestore =  false;
+
+    UserService.get().then((user) => {
+        if(user.permissions && user.permissions.restoreContent && user.permissions.restoreContent === true) {
+          $scope.canRestore = true;
+        }
+    }).catch ((err) => {
+        //send the error via the mediator
+        console.log('error', err);
```
Comment (hmgibson23) [inline public/javascripts/app/controllers/SnapshotContentCtrl.js]: Think this was left behind...

**Comment by hmgibson23** [comment]
👍

**Comment by na7hangood** [comment]
Played in CODE. Looks great and permissions work as expected.

File: public/javascripts/app/controllers/SnapshotContentCtrl.js (hunk 1 above)
Comment (susiecoleman) [inline public/javascripts/app/controllers/SnapshotContentCtrl.js]: This is used in the restore-list file to hide and show the restore button

Comment (susiecoleman) [inline public/javascripts/app/controllers/SnapshotContentCtrl.js]: Fixed in this PR to use it in logging https://github.com/guardian/composer-restorer/pull/90

**Pull request merged**